src/app/api/gql/fragments/fragment-metadata-block.graphql

```diff
@@ -18,5 +18,8 @@ fragment MetadataBlock on MetadataBlockExtended {
     ...DatasetCurrentInfo
     ...SetLicenseEvent
     ...SetPollingSourceEvent
+    ...AddPushSourceEvent
+    ...SetDataSchemaEvent
+    ...DisablePollingSourceEvent
```

Comment:
Disable push source too?

Reply:
I'll add it when I do the metadata tab with editing